SpriterDotNet/Spriter.cs

```diff
@@ -266,17 +211,13 @@ public abstract class SpriterKey : SpriterElement
 
         [XmlAttribute("c4")]
         public float C4;
-
-        public SpriterKey()
-        {
-            Time = 0;
-            CurveType = SpriterCurveType.Linear;
-        }
     }
 
     [Serializable]
     public enum SpriterObjectType
     {
+        None = 0,
```

Comment:
If I understand correctly, timelines should always have an object type - even if it's an implicit default value. In other cases where the value of an enum might really be absent, it makes perfect sense to have a none value or even better, a nullable enum (http://stackoverflow.com/questions/1795657/c-sharp-enums-nullable-or-unknown-value). So in this case I'd prefer to stick with the enum as it is.